pkg/webui/components/qr/qr.styl

```
@@ -0,0 +1,25 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.wrapper
+  width: 100%
+  aspect-ratio: 4 / 3
+  background: $c-backdrop-dark
```

```suggestion
  background: 'black'
```
I think this color is a special case as backdrop for an empty camera screen that does not need an extra variable.